src/routing/core/messaging/handling/mod.rs

```diff
@@ -53,13 +53,9 @@ impl Core {
 
         // Check if the message is for us.
         let in_dst_location = msg.dst().contains(&self.node.name(), self.section.prefix());
+        // TODO: Broadcast message to our section of src is a Node.
```

Comment:
better expand the comment little bit to explain why need to do so.
also
`to our section of src is a Node` => `to our section when src is a Node`